### ISSUE TITLE
With the example AMIs, doesn't install the php / apache module, and thus the ganglia Web doesn't work

### ISSUE THREAD

**Issue by guarly** [nathanmarz/storm-deploy#25]
I've solved it with the following code change:

--- a/src/clj/backtype/storm/crate/ganglia.clj
+++ b/src/clj/backtype/storm/crate/ganglia.clj
@@ -17,7 +17,7 @@
   [session]
   (-> session
       (package/packages
-       :aptitude ["rrdtool" "librrds-perl" "librrd-dev" "php5-gd"
-       :aptitude ["rrdtool" "librrds-perl" "librrd-dev" "php5-gd" "libapache2-mod-php5"
                 "ganglia-monitor" "ganglia-webfrontend" "gmetad"])
     (file/symbolic-link
      "/usr/share/ganglia-webfrontend" "/var/www/ganglia")))

**Comment by guarly** [comment]
Sorry:

```
--- a/src/clj/backtype/storm/crate/ganglia.clj
+++ b/src/clj/backtype/storm/crate/ganglia.clj
@@ -17,7 +17,7 @@
   [session]
   (-> session
       (package/packages
-       :aptitude ["rrdtool" "librrds-perl" "librrd-dev" "php5-gd"
+       :aptitude ["rrdtool" "librrds-perl" "librrd-dev" "php5-gd" "libapache2-mod-php5"
                   "ganglia-monitor" "ganglia-webfrontend" "gmetad"])
       (file/symbolic-link
        "/usr/share/ganglia-webfrontend" "/var/www/ganglia")))
```

**Comment by lorcan** [comment]
This issue was resolved with 580d97bb8a5d446f3cba4d06d1f509732a7227af. This issue should be closed.
